libr/bin/format/elf/elf.c

```diff
@@ -2118,8 +2196,26 @@ char* Elf_(r_bin_elf_get_head_flag)(ELFOBJ *bin) {
 		default :                   return strdup (" Unknown mips ISA");
 		}
 	}
-	//TODO: Fill other arch 
-	return strdup("unknown_flag");
+	return NULL;
+}
+
+char* Elf_(r_bin_elf_get_head_flag)(ELFOBJ *bin) {
```

Comment:
The information parsed by this function is currently not used, should I delete this function or keep it but nothing is done in the function

Reply:
I think this is used in some cases. So I'd keep it.